src/index.html

```diff
@@ -22,6 +22,393 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="header__logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo"
+          class="header__logo-img"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a
+              href="#"
+              class="header__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__item">
```

Comment:
Add the blank line between the links 
```suggestion
          </li>
          
          <li class="header__item">
```